doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -547,7 +547,7 @@ Linear Algebra Equivalents
      - eigenvalues and eigenvectors of ``a``
 
    * - ``[V,D]=eig(a,b)``
-     - ``V,D = np.linalg.eig(a,b)``
+     - ``D,V = scipy.linalg.eig(a,b)``
```

Comment:
No need for `scipy.linalg`, `np.linalg` has that function. Hmm, the docstring of `np.linalg.eig` could use some improvement.

Reply:
@charris the function is there, but its signature only accepts a single array, i.e. it doesn't support _generalized_ eigenproblems.

Reply:
OK.